packages/code-du-travail-frontend/src/conventions/ConventionListeTextes.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import getConfig from "next/config";
+import ConventionTexte from "./ConventionTexte";
+
+const {
+  publicRuntimeConfig: { API_DILA2SQL_URL }
+} = getConfig();
+
+class ConventionListeTextes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { loaded: false, textes: [], selectedTexte: null };
+  }
+
+  fetchTextes({ conteneurId, typeTextes }) {
```

Comment:
this should be in a `convention.service.js`

Reply:
good call, I'll move it 👍